doc/pyapi/index.rst

```diff
@@ -329,8 +329,8 @@ Power iteration solver
    :template: python.rst
 
    solver.PowerIterationKEigenSolver
-   solver.PowerIterationKEigenSCDSASolver
-   solver.PowerIterationKEigenSMMSolver
+   solver.PowerIterationKEigenSCDSASolver TODO FIX ME
```

Comment:
I'm not sure what I need to update here

Reply:
Maybe schedule a discussion with @wdhawkins and or with me.